src/main/java/de/rabea/game/GameSetup.java

```diff
@@ -15,17 +15,28 @@ public GameSetup(UserInterface userInterface, PlayerFactory playerFactory) {
 
     public void startApplication() {
         userInterface.greet();
-        setUpNewGame();
+        setUpGame();
     }
 
-    public void setUpNewGame() {
+    public void setUpGame() {
```

Comment:
Thanks for the comments, @christophgockel! I tried to improve the naming with my last commit.